### PR TITLE
Fixing the AudioWorklet Recorder download button link

### PR DESCRIPTION
**Pull request by TenghuiZhang**
Fixing #298 The download button is not functional

### CODE REVIEW

File: src/audio-worklet/migration/worklet-recorder/app.js [hunk 1]
```diff
@@ -327,7 +327,7 @@ const createRecord = (recordingProperties, recordingLength, sampleRate,
   const recordingUrl = createLinkFromAudioBuffer(recordingBuffer, true);
 
   player.src = recordingUrl;
-  downloadLink.src = recordingUrl;
+  downloadLink.href = recordingUrl;
```
Comment (hoch) [inline src/audio-worklet/migration/worklet-recorder/app.js:330]: Hah! That's a nice catch. Do you have a staging page link from your fork?

Comment (TenghuiZhang) [reply src/audio-worklet/migration/worklet-recorder/app.js:330]: No, I didn't. And I am wondering why "Build and deploy to GitHub Pages / Build and deploy to GitHub Pages (pull_request)" It seems like I can not pull request to the main branch due to the deploy key?

Comment (hoch) [reply src/audio-worklet/migration/worklet-recorder/app.js:330]: If the auto deploy is not possible, could you locally check if the patch fixes the issue? (I believe you already did, but just to double check)

Comment (TenghuiZhang) [reply src/audio-worklet/migration/worklet-recorder/app.js:330]: Yes, the issue is fixed

**Comment by TenghuiZhang** [comment]
The github pre-checking block is passed with the newest permission setup

**Pull request merged**